<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpcodeFinderClass</class>
 <widget class="QWidget" name="OpcodeFinderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>331</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Silkroad Opcode Finder - ProjectHax.com</string>
  </property>
  <property name="windowIcon">
   <iconset resource="opcodefinder.qrc">
    <normaloff>:/OpcodeFinder/PH.png</normaloff>:/OpcodeFinder/PH.png</iconset>
  </property>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QLineEdit" name="txtPath1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPath2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstServer1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstClient1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstClient2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstServer2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="opcodefinder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOpen1</sender>
   <signal>clicked()</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpen2</sender>
   <signal>clicked()</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstServer1</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>RowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstClient1</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>RowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstServer2</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>RowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstClient2</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>OpcodeFinderClass</receiver>
   <slot>RowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
  <slot>RowChanged(int)</slot>
 </slots>
</ui>
